# ru.po
# LIBPQ Translated Messages into the Russian Language (UTF-8)
#
# Copyright (c) 2001-2004 Serguei A. Mokhov, mokhov@cs.concordia.ca
# Distributed under the same licensing terms as PostgreSQL itself.
#
# pgtranslation Id: libpq.po,v 1.4 2010/12/07 21:22:21 petere Exp $
#
# ChangeLog:
#   - November, 2010: Updates for PostgreSQL 9 by Andrey Sudnik <sudnikand@yandex.ru>
#   - January, 2005: Corrections and improvements by Oleg Bartunov <oleg@sai.msu.su>
#   - March 27 - November 1, 2004: Updates for 8.0; <mokhov@cs.concordia.ca>
#   - July 24 - October 5, 2003: Updates for 7.4.*; <mokhov@cs.concordia.ca>
#   - Januray 2, 2002: Completed SSL translation, Serguei A. Mokhov <mokhov@cs.concordia.ca>
#   - October 12, 2002: Post-7.3beta2 fuzzy translation fixes, Serguei A. Mokhov <mokhov@cs.concordia.ca>
#   - September 7, 2002: Complete post-7.3beta1 Translation, Serguei A. Mokhov <mokhov@cs.concordia.ca>
#   - August 2001 - August 2002: Initial translation and maintenance, Serguei A. Mokhov <mokhov@cs.concordia.ca>
#
msgid ""
msgstr ""
"Project-Id-Version: PostgreSQL 9 current\n"
"Report-Msgid-Bugs-To: pgsql-bugs@postgresql.org\n"
"POT-Creation-Date: 2010-11-29 18:43+0000\n"
"PO-Revision-Date: 2010-11-30 20:14+0600\n"
"Last-Translator: Andrey Sudnik <sudnikand@yandex.ru>\n"
"Language-Team: pgsql-rus\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-Language: Russian\n"
"X-Poedit-Country: RUSSIAN FEDERATION\n"
"X-Poedit-SourceCharset: utf-8\n"

#: fe-auth.c:212
#: fe-auth.c:432
#: fe-auth.c:659
msgid "host name must be specified\n"
msgstr "должно быть указано имя хоста\n"

#: fe-auth.c:242
#, c-format
msgid "could not set socket to blocking mode: %s\n"
msgstr "не удалось установить сокет в блокирующий режим: %s\n"

#: fe-auth.c:260
#: fe-auth.c:264
#, c-format
msgid "Kerberos 5 authentication rejected: %*s\n"
msgstr "аутентификация Kerberos 5 отклонена: %*s\n"

#: fe-auth.c:290
#, c-format
msgid "could not restore non-blocking mode on socket: %s\n"
msgstr "не удалось восстановить неблокирующий режим для сокета: %s\n"

#: fe-auth.c:403
msgid "GSSAPI continuation error"
msgstr "GSSAPI ошибка возобновления"

#: fe-auth.c:439
msgid "duplicate GSS authentication request\n"
msgstr "повторный запрос аутентификации GSS\n"

#: fe-auth.c:459
msgid "GSSAPI name import error"
msgstr "GSSAPI ошибка импорта имени"

#: fe-auth.c:545
msgid "SSPI continuation error"
msgstr "SSPI ошибка возобновления"

#: fe-auth.c:556
#: fe-auth.c:630
#: fe-auth.c:665
#: fe-auth.c:762
#: fe-connect.c:1970
#: fe-connect.c:3369
#: fe-connect.c:3586
#: fe-connect.c:4002
#: fe-connect.c:4011
#: fe-connect.c:4148
#: fe-connect.c:4194
#: fe-connect.c:4212
#: fe-connect.c:4291
#: fe-connect.c:4361
#: fe-connect.c:4407
#: fe-connect.c:4425
#: fe-exec.c:3121
#: fe-exec.c:3286
#: fe-lobj.c:696
#: fe-protocol2.c:1027
#: fe-protocol3.c:1425
msgid "out of memory\n"
msgstr "недостаточно памяти\n"

#: fe-auth.c:645
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить SSPI удостоверение"

#: fe-auth.c:738
msgid "SCM_CRED authentication method not supported\n"
msgstr "аутентификация SCM_CRED не поддерживается\n"

#: fe-auth.c:812
msgid "Kerberos 4 authentication not supported\n"
msgstr "аутентификация Kerberos 4 не поддерживается\n"

#: fe-auth.c:828
msgid "Kerberos 5 authentication not supported\n"
msgstr "аутентификация Kerberos 5 не поддерживается\n"

#: fe-auth.c:895
msgid "GSSAPI authentication not supported\n"
msgstr "аутентификация GSSAPI не поддерживается\n"

#: fe-auth.c:919
msgid "SSPI authentication not supported\n"
msgstr "аутентификация SSPI не поддерживается\n"

#: fe-auth.c:926
msgid "Crypt authentication not supported\n"
msgstr "Шифрованая аутентификация не поддерживается\n"

#: fe-auth.c:953
#, c-format
msgid "authentication method %u not supported\n"
msgstr "метод аутентификации %u не поддерживается\n"

#: fe-connect.c:759
#, c-format
msgid "invalid sslmode value: \"%s\"\n"
msgstr "неверное значение sslmode: \"%s\"\n"

#: fe-connect.c:780
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "значение sslmode \"%s\" неверно, если поддержка SSL не скомпилирована\n"

#: fe-connect.c:963
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "не удалось установить сокет в режим TCP передачи без задержки: %s\n"

#: fe-connect.c:993
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running locally and accepting\n"
"\tconnections on Unix domain socket \"%s\"?\n"
msgstr ""
"не удалось соединиться с сервером: %s\n"
"\tРаботает ли сервер локально и принимает\n"
"\tсоединения через сокет Unix-домена \"%s\"?\n"

#: fe-connect.c:1037
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\"%s%s%s and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"не удалось подсоединиться к серверу: %s\n"
"\tРаботает ли сервер на хосте \"%s\"%s%s%s и принимает ли\n"
"\tTCP/IP соединения по порту %s?\n"

#: fe-connect.c:1096
#, c-format
msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPIDLE) не удался: %s\n"

#: fe-connect.c:1109
#, c-format
msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPALIVE) не удался: %s\n"

#: fe-connect.c:1141
#, c-format
msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
msgstr "setsockopt(TCP_KEEPINTVL) не удался: %s\n"

#: fe-connect.c:1173
#, c-format
msgid "setsockopt(TCP_KEEPCNT) failed: %s\n"
msgstr "setsockopt(TCP_KEEPCNT) не удался: %s\n"

#: fe-connect.c:1222
#, c-format
msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
msgstr "WSAIoctl(SIO_KEEPALIVE_VALS) потерпел неудачу: %ui\n"

#: fe-connect.c:1274
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "неверный номер порта: \"%s\"\n"

#: fe-connect.c:1317
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "не удалось преобразовать имя хоста \"%s\" в адрес: %s\n"

#: fe-connect.c:1321
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr "не удалось преобразовать название директории сокетов UNIX-домена \"%s\" в адрес: %s\n"

#: fe-connect.c:1531
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения, возможный указатель на нарушение целостности памяти\n"

#: fe-connect.c:1572
#, c-format
msgid "could not create socket: %s\n"
msgstr "не удалось создать сокет: %s\n"

#: fe-connect.c:1595
#, c-format
msgid "could not set socket to non-blocking mode: %s\n"
msgstr "не удалось установить сокет в неблокирующий режим: %s\n"

#: fe-connect.c:1607
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr "не удалось установить сокет в режим закрытия-на-выполнение (close-on-exec): %s\n"

#: fe-connect.c:1625
msgid "keepalives parameter must be an integer\n"
msgstr "keepalives параметр должен быть целым числом\n"

#: fe-connect.c:1638
#, c-format
msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(SO_KEEPALIVE) не удался: %s\n"

#: fe-connect.c:1779
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "не удалось получить статус ошибки сокета: %s\n"

#: fe-connect.c:1817
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "не удалось получить адрес клиента из сокета: %s\n"

#: fe-connect.c:1849
#: fe-connect.c:1863
#: fe-connect.c:1875
#, c-format
msgid "could not get peer credentials: %s\n"
msgstr "не удалось получить удостоверение peer-a: %s\n"

#: fe-connect.c:1883
#, c-format
msgid "could not get effective UID from peer credentials: %s\n"
msgstr "не удалось получить эффективный UID из удостоверения peer-а: %s\n"

#: fe-connect.c:1898
#, c-format
msgid "local user with ID %d does not exist\n"
msgstr "локальный пользователь с ID %d не существует\n"

#: fe-connect.c:1906
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\"\n"
msgstr "requirepeer задан как \"%s\", но настоящее имя пользователя peer-а \"%s\"\n"

#: fe-connect.c:1912
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "параметр requirepeer не поддерживаются на этой платформе\n"

#: fe-connect.c:1944
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования SSL: %s\n"

#: fe-connect.c:1983
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "не удалось отправить начальный пакет: %s\n"

#: fe-connect.c:2050
#: fe-connect.c:2069
msgid "server does not support SSL, but SSL was required\n"
msgstr "сервер не поддерживает SSL, но наличие SSL было необходимо\n"

#: fe-connect.c:2085
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "получен неверный ответ на согласование по SSL: %c\n"

#: fe-connect.c:2161
#: fe-connect.c:2194
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "ожидался запрос аутентификации от сервера, но было получено: %c\n"

#: fe-connect.c:2367
#, c-format
msgid "out of memory allocating GSSAPI buffer (%i)"
msgstr "недостаточно памяти для аллоцирования GSSAPI буфера (%i)"

#: fe-connect.c:2452
msgid "unexpected message from server during startup\n"
msgstr "неожиданное сообщение от сервера при запуске\n"

#: fe-connect.c:2548
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения %d, возможно, это признак нарушения целостности памяти\n"

#: fe-connect.c:2977
#: fe-connect.c:3037
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "PGEventProc \"%s\" потерпел неудачу во время события PGEVT_CONNRESET\n"

#: fe-connect.c:3382
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "некорректный LDAP URL \"%s\": схема должна быть ldap://\n"

#: fe-connect.c:3397
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "не корректный LDAP URL \"%s\": отсутствует уникальное имя\n"

#: fe-connect.c:3408
#: fe-connect.c:3461
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "не корректный LDAP URL \"%s\": должен быть только один аттрибут\n"

#: fe-connect.c:3418
#: fe-connect.c:3475
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr "не корректный LDAP URL \"%s\": должна быть границы поиска (base/one/sub)\n"

#: fe-connect.c:3429
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "не корректный LDAP URL \"%s\": нет фильтра\n"

#: fe-connect.c:3450
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "не корректный LDAP URL \"%s\": неверный номер порта\n"

#: fe-connect.c:3484
msgid "could not create LDAP structure\n"
msgstr "не удалось создать LDAP структуру\n"

#: fe-connect.c:3526
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "поиск на LDAP сервере закончился неудачно: %s\n"

#: fe-connect.c:3537
msgid "more than one entry found on LDAP lookup\n"
msgstr "более одного вхождения обнаружено при LDAP поиске\n"

#: fe-connect.c:3538
#: fe-connect.c:3550
msgid "no entry found on LDAP lookup\n"
msgstr "ни одного вхождения не обнаружено при LDAP поиске\n"

#: fe-connect.c:3561
#: fe-connect.c:3574
msgid "attribute has no values on LDAP lookup\n"
msgstr "аттрибут не имеет значения при LDAP поиске\n"

#: fe-connect.c:3625
#: fe-connect.c:3643
#: fe-connect.c:4050
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "отсутствует \"=\" после \"%s\" в строке соединения\n"

#: fe-connect.c:3706
#: fe-connect.c:4132
#: fe-connect.c:4316
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "неверный параметр соединения \"%s\"\n"

#: fe-connect.c:3719
#: fe-connect.c:4099
msgid "unterminated quoted string in connection info string\n"
msgstr "неоконченная строка (не хватает кавычки) в строке соединения\n"

#: fe-connect.c:3758
msgid "could not get home directory to locate service definition file"
msgstr "не удалось получить домашнюю директорию, чтобы найти файл со служебными определениями"

#: fe-connect.c:3791
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "определение сервиса \"%s\" не найдено\n"

#: fe-connect.c:3814
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "служебный файл \"%s\" не найден\n"

#: fe-connect.c:3827
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "строка %d слишком длинная в служебном файле \"%s\"\n"

#: fe-connect.c:3898
#: fe-connect.c:3925
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "синтаксическая ошибка в служебном файле \"%s\", строка %d\n"

#: fe-connect.c:4592
msgid "connection pointer is NULL\n"
msgstr "указатель соединения является NULL\n"

#: fe-connect.c:4865
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "ВНИМАНИЕ: файл паролей \"%s\" не обычный текстовый файл\n"

#: fe-connect.c:4874
#, c-format
msgid "WARNING: password file \"%s\" has group or world access; permissions should be u=rw (0600) or less\n"
msgstr "ВНИМАНИЕ: файл с паролями \"%s\" имеет права на чтение для всех или для группы; права должны быть u=rw (0600) или меньше\n"

#: fe-connect.c:4962
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "пароль получен из файла \"%s\"\n"

#: fe-exec.c:827
msgid "NOTICE"
msgstr "NOTICE"

#: fe-exec.c:1014
#: fe-exec.c:1071
#: fe-exec.c:1111
msgid "command string is a null pointer\n"
msgstr "командная строка является нулевым указателем\n"

#: fe-exec.c:1104
#: fe-exec.c:1199
msgid "statement name is a null pointer\n"
msgstr "имя команды является нулевым указателем\n"

#: fe-exec.c:1119
#: fe-exec.c:1273
#: fe-exec.c:1928
#: fe-exec.c:2125
msgid "function requires at least protocol version 3.0\n"
msgstr "функция требует как минимум версию протокола 3.0\n"

#: fe-exec.c:1230
msgid "no connection to the server\n"
msgstr "нет соединения с сервером\n"

#: fe-exec.c:1237
msgid "another command is already in progress\n"
msgstr "другая команда уже выполняется\n"

#: fe-exec.c:1349
msgid "length must be given for binary parameter\n"
msgstr "для бинарного параметра должа быть указана длина\n"

#: fe-exec.c:1596
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "неожиданный asyncStatus: %d\n"

#: fe-exec.c:1616
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "PGEventProc \"%s\" порушился во время события PGEVT_RESULTCREATE\n"

#: fe-exec.c:1746
msgid "COPY terminated by new PQexec"
msgstr "COPY прекращена новой PQexec"

#: fe-exec.c:1754
msgid "COPY IN state must be terminated first\n"
msgstr "Сначала, состояние COPY IN должно завершиться\n"

#: fe-exec.c:1774
msgid "COPY OUT state must be terminated first\n"
msgstr "Сначала, состояние COPY OUT должно завершиться\n"

#: fe-exec.c:2016
#: fe-exec.c:2082
#: fe-exec.c:2167
#: fe-protocol2.c:1172
#: fe-protocol3.c:1561
msgid "no COPY in progress\n"
msgstr "COPY не выполняется\n"

#: fe-exec.c:2359
msgid "connection in wrong state\n"
msgstr "соединение в неправильном состоянии\n"

#: fe-exec.c:2390
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:2454
#: fe-exec.c:2477
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "номер колонки %d вне диапазона 0..%d"

#: fe-exec.c:2470
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "номер записи %d вне диапазона 0..%d"

#: fe-exec.c:2492
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "номер параметра %d вне диапазона 0..%d"

#: fe-exec.c:2780
#, c-format
msgid "could not interpret result from server: %s"
msgstr "не в состоянии интерпретировать ответ сервера: %s"

#: fe-exec.c:3019
#: fe-exec.c:3103
msgid "incomplete multibyte character\n"
msgstr "неполный мультибайтовый символ\n"

#: fe-lobj.c:152
msgid "cannot determine OID of function lo_truncate\n"
msgstr "не в состоянии определить OID функции lo_truncate\n"

#: fe-lobj.c:380
msgid "cannot determine OID of function lo_create\n"
msgstr "не в состоянии определить OID функции lo_create\n"

#: fe-lobj.c:525
#: fe-lobj.c:624
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "не в состоянии открыть файл \"%s\": %s\n"

#: fe-lobj.c:575
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "не в состоянии произвести чтение из файла \"%s\": %s\n"

#: fe-lobj.c:639
#: fe-lobj.c:663
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "не в состоянии призвести запись в файл \"%s\": %s\n"

#: fe-lobj.c:744
msgid "query to initialize large object functions did not return data\n"
msgstr "запрос об инициализации функций для больших объектов (LOs) не вернул данных\n"

#: fe-lobj.c:785
msgid "cannot determine OID of function lo_open\n"
msgstr "не в состоянии определить OID функции lo_open\n"

#: fe-lobj.c:792
msgid "cannot determine OID of function lo_close\n"
msgstr "не в состоянии определить OID функции lo_close\n"

#: fe-lobj.c:799
msgid "cannot determine OID of function lo_creat\n"
msgstr "не в состоянии определить OID функции lo_creat\n"

#: fe-lobj.c:806
msgid "cannot determine OID of function lo_unlink\n"
msgstr "не в состоянии определить OID функции lo_unlink\n"

#: fe-lobj.c:813
msgid "cannot determine OID of function lo_lseek\n"
msgstr "не в состоянии определить OID функции lo_lseek\n"

#: fe-lobj.c:820
msgid "cannot determine OID of function lo_tell\n"
msgstr "не в состоянии определить OID функции lo_tell\n"

#: fe-lobj.c:827
msgid "cannot determine OID of function loread\n"
msgstr "не в состоянии определить OID функции loread\n"

#: fe-lobj.c:834
msgid "cannot determine OID of function lowrite\n"
msgstr "не в состоянии определить OID функции lowrite\n"

#: fe-misc.c:262
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "тип integer длиной %lu байт не поддерживается функцией pqGetInt"

#: fe-misc.c:298
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "тип integer длиной %lu байт не поддерживается функцией pqPutInt"

#: fe-misc.c:578
#: fe-misc.c:780
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:643
#: fe-misc.c:733
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "ошибка при получении данных с сервера: %s\n"

#: fe-misc.c:750
#: fe-misc.c:828
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"сервер неожиданно закрыл соединение\n"
"\tСкорее всего это означает, что сервер завершил работу со сбоем\n"
"\tдо или в процессе выполнения запроса.\n"

#: fe-misc.c:845
#, c-format
msgid "could not send data to server: %s\n"
msgstr "не удалось послать данные серверу: %s\n"

#: fe-misc.c:964
msgid "timeout expired\n"
msgstr "интервал ожидания закончился\n"

#: fe-misc.c:1009
msgid "socket not open\n"
msgstr "сокет не открыт\n"

#: fe-misc.c:1032
#, c-format
msgid "select() failed: %s\n"
msgstr "select() не удался: %s\n"

#: fe-protocol2.c:89
#, c-format
msgid "invalid setenv state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние setenv %c, возможный указатель на нарушение целостности памяти\n"

#: fe-protocol2.c:330
#, c-format
msgid "invalid state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние %c, возможный указатель на нарушение целостности памяти\n"

#: fe-protocol2.c:419
#: fe-protocol3.c:186
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "сообщение типа 0x%02x прибыло от сервера во время простоя"

#: fe-protocol2.c:462
#, c-format
msgid "unexpected character %c following empty query response (\"I\" message)"
msgstr "неожиданный символ %c вслед за пустым ответом (сообщение \"I\")"

#: fe-protocol2.c:516
msgid "server sent data (\"D\" message) without prior row description (\"T\" message)"
msgstr "сервер отправил данные (сообщение \"D\") без предварительного строки описания (сообщение \"T\")"

#: fe-protocol2.c:532
msgid "server sent binary data (\"B\" message) without prior row description (\"T\" message)"
msgstr "сервер отправил двоичные данные (сообщение \"B\") без предварительного строки описания (сообщение \"T\")"

#: fe-protocol2.c:547
#: fe-protocol3.c:382
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\"\n"

#: fe-protocol2.c:768
#: fe-protocol3.c:701
msgid "out of memory for query result\n"
msgstr "недостаточно памяти для результата запроса\n"

#: fe-protocol2.c:1215
#: fe-protocol3.c:1629
#, c-format
msgid "%s"
msgstr "%s"

#: fe-protocol2.c:1227
msgid "lost synchronization with server, resetting connection"
msgstr "потеряна синхронизация с сервером; переустановка соединения"

#: fe-protocol2.c:1361
#: fe-protocol2.c:1393
#: fe-protocol3.c:1832
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "ошибка протокола: id=0x%x\n"

#: fe-protocol3.c:344
msgid "server sent data (\"D\" message) without prior row description (\"T\" message)\n"
msgstr "сервер отправил данные (сообщение \"D\") без предварительного строки описания (сообщение \"T\")\n"

#: fe-protocol3.c:403
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "содержимое сообщения не согласуется с длиной в сообщении типа \"%c\"\n"

#: fe-protocol3.c:424
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr "потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d\n"

#: fe-protocol3.c:646
msgid "unexpected field count in \"D\" message\n"
msgstr "неожиданное число полей в собщении \"D\"\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:792
#: fe-protocol3.c:811
#, c-format
msgid " at character %s"
msgstr " у символа %s"

#: fe-protocol3.c:824
#, c-format
msgid "DETAIL:  %s\n"
msgstr "ПОДРОБНО:  %s\n"

#: fe-protocol3.c:827
#, c-format
msgid "HINT:  %s\n"
msgstr "ПОДСКАЗКА:  %s\n"

#: fe-protocol3.c:830
#, c-format
msgid "QUERY:  %s\n"
msgstr "ЗАПРОС:  %s\n"

#: fe-protocol3.c:833
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "КОНТЕКСТ:  %s\n"

#: fe-protocol3.c:845
msgid "LOCATION:  "
msgstr "РАСПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:847
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:849
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1073
#, c-format
msgid "LINE %d: "
msgstr "СТРОКА %d:"

#: fe-protocol3.c:1457
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline: не производится текстовый COPY OUT\n"

#: fe-secure.c:265
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "не удалось установить SSL-соединение: %s\n"

#: fe-secure.c:349
#: fe-secure.c:436
#: fe-secure.c:1162
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "ошибка SSL SYSCALL: %s\n"

#: fe-secure.c:355
#: fe-secure.c:442
#: fe-secure.c:1166
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "ошибка SSL SYSCALL: обнаружен конец файла (EOF)\n"

#: fe-secure.c:367
#: fe-secure.c:453
#: fe-secure.c:1175
#, c-format
msgid "SSL error: %s\n"
msgstr "ошибка SSL: %s\n"

#: fe-secure.c:377
#: fe-secure.c:463
#: fe-secure.c:1184
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "нераспознанный код ошибки SSL: %d\n"

#: fe-secure.c:601
msgid "host name must be specified for a verified SSL connection\n"
msgstr "для заверенного SSL соединения должно быть указано имя хоста\n"

#: fe-secure.c:620
#, c-format
msgid "server common name \"%s\" does not match host name \"%s\"\n"
msgstr "общее имя сервера \"%s\" не совпадает с именем хоста \"%s\"\n"

#: fe-secure.c:752
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "не удалось создать контекст SSL: %s\n"

#: fe-secure.c:843
msgid "could not get home directory to locate client certificate files\n"
msgstr "не удалось получить домашнюю директорию, чтобы найти файлы с клиентскими сертификатами\n"

#: fe-secure.c:868
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "не в состоянии открыть файл сертификата \"%s\": %s\n"

#: fe-secure.c:893
#: fe-secure.c:903
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "не в состоянии считать файл сертификата \"%s\": %s\n"

#: fe-secure.c:940
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "не удаётся загрузить движок SSL \"%s\": %s\n"

#: fe-secure.c:952
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "не удаётся инициализировать движок SSL \"%s\": %s\n"

#: fe-secure.c:968
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось прочитать приватный SSL ключ \"%s\" из движка \"%s\": %s\n"

#: fe-secure.c:982
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось загрузить приватный SSL ключ \"%s\" из движка \"%s\": %s\n"

#: fe-secure.c:1017
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "сертификат присутствует, но нет файла с личным ключом \"%s\"\n"

#: fe-secure.c:1025
#, c-format
msgid "private key file \"%s\" has group or world access; permissions should be u=rw (0600) or less\n"
msgstr "файл приватного ключа \"%s\" имеет права на чтение для всех или для группы; права должны быть u=rw (0600) или меньше\n"

#: fe-secure.c:1036
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "не удалось загрузить файл приватного ключа \"%s\": %s\n"

#: fe-secure.c:1050
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "сертификат не совпадает с файлом личного ключа \"%s\": %s\n"

#: fe-secure.c:1075
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл корневых сертификатов \"%s\": %s\n"

#: fe-secure.c:1099
#, c-format
msgid "SSL library does not support CRL certificates (file \"%s\")\n"
msgstr "Библиотека SSL не поддерживает CRL сертификаты (файл \"%s\")\n"

#: fe-secure.c:1120
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate verification.\n"
msgstr ""
"корневой файл сертификата \"%s\" не существует\n"
"Предоставьте файл с сертификатом или измените sslmode, чтобы отключить проверку сертификата.\n"

#: fe-secure.c:1203
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "не удалось получить сертификат: %s\n"

#: fe-secure.c:1231
msgid "SSL certificate's common name contains embedded null\n"
msgstr "Общее имя SSL сертификата содержит вставленный null\n"

#: fe-secure.c:1307
msgid "no SSL error reported"
msgstr "нераспознанный код ошибки SSL"

#: fe-secure.c:1316
#, c-format
msgid "SSL error code %lu"
msgstr "ошибка SSL: %lu"

#~ msgid "error querying socket: %s\n"
#~ msgstr "ошибка при запросе сокета: %s\n"

#~ msgid "could not get information about host \"%s\": %s\n"
#~ msgstr "не удалось получить информацию о хосте \"%s\": %s\n"

#~ msgid "unsupported protocol\n"
#~ msgstr "протокол не поддерживается\n"

#~ msgid "server common name \"%s\" does not resolve to %ld.%ld.%ld.%ld\n"
#~ msgstr "имя сервера \"%s\" не транслируется в %ld.%ld.%ld.%ld\n"

#~ msgid "could not get user information\n"
#~ msgstr "не удалось получить информацию о пользователе\n"

#~ msgid "invalid value of PGSSLKEY environment variable\n"
#~ msgstr "неверное значение для переменной окружения PGSSLKEY\n"

#~ msgid "private key file \"%s\" has wrong permissions\n"
#~ msgstr "неправильные права доступа к  файла личного ключа \"%s\"\n"

#~ msgid "could not open private key file \"%s\": %s\n"
#~ msgstr "не удалось открыть файл личного ключа \"%s\": %s\n"

#~ msgid "private key file \"%s\" changed during execution\n"
#~ msgstr "личный ключ \"%s\" изменился во время исполнения\n"

#~ msgid "certificate could not be validated: %s\n"
#~ msgstr "сертификат не может быть проверен: %s\n"

#~ msgid "Kerberos 4 error: %s\n"
#~ msgstr "ошибка Kerberos 4: %s\n"

#~ msgid "Kerberos 4 authentication failed\n"
#~ msgstr "аутентификация Kerberos 4 не удалась\n"

#~ msgid "Kerberos 5 authentication failed\n"
#~ msgstr "аутентификация Kerberos 5 не удалась\n"

#~ msgid "invalid authentication service name \"%s\", ignored\n"
#~ msgstr "неверное имя сервиса аутентификации \"%s\", игнорируется\n"

#~ msgid "fe_getauthname: invalid authentication system: %d\n"
#~ msgstr "fe_getauthname: неверная система аутентификации: %d\n"

#~ msgid "could not receive server response to SSL negotiation packet: %s\n"
#~ msgstr "не был получен ответ сервера на пакет согласования SSL: %s\n"

#~ msgid "could not create large object for file \"%s\"\n"
#~ msgstr "не в состоянии создать большой объект (LO) для файла \"%s\"\n"

#~ msgid "could not open large object %u\n"
#~ msgstr "не в состоянии открыть большой объект (LO) %u\n"

#~ msgid "error while reading file \"%s\"\n"
#~ msgstr "ошибка при чтении файла \"%s\"\n"

#~ msgid "error while writing to file \"%s\"\n"
#~ msgstr "ошибка при записи в файл \"%s\"\n"
